instrumentation/runtime-metrics/library/src/test/java/io/opentelemetry/instrumentation/runtimemetrics/BufferPoolsTest.java

```diff
@@ -36,6 +46,74 @@ void setup() {
     beans = Arrays.asList(bufferPoolBean);
   }
 
+  @Test
+  void registerObservers() {
```

Comment:
Also adding some test coverage to make this consistent with the other runtime observers.